bin/gem2rpm

```diff
@@ -26,11 +26,33 @@ end
 
 rest = options[:args]
 
-template = begin
-  Gem2Rpm::Template.find options[:template_file], :gem_file => rest[0]
-rescue Gem2Rpm::Template::TemplateError => e
-  $stderr.puts e
-  exit(1)
+template = nil
+if template_file.nil?
```

Comment:
Where does this variable come from? If it isn't defined, lower inside the block it'll be a local variable which means the code doesn't work.

Reply:
I forgot to add it from `options`. Thanks for checking!